docker/Dockerfile.dev

```diff
@@ -0,0 +1,13 @@
+FROM esmvalgroup/esmvalcore:latest
+
+COPY . /esmvaltool
+WORKDIR /esmvaltool
+
+RUN conda env update -n base environment.yml
+RUN pip install .[develop]
+RUN Rscript esmvaltool/install/R/setup.R
+RUN julia esmvaltool/install/Julia/setup.jl
+
+# Clean ESMValTool
+RUN pip uninstall esmvaltool -y
```

Comment:
Why are you removing esmvaltool again?

Reply:
Ah, I get it, to just get the dependencies for testing, right?

Reply:
Yes